TestAssets/TestProjects/TestAppWithSigning/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "emitEntryPoint": true,
+    "keyFile": "./key.snk"
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.1"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {}
+  },
+  "runtimes": {
+    "win7-x64": {},
```

Comment:
Any reason why this needs to be a self-contained app?  Portable would be more simple, and wouldn't need to restore the runtime assets.